process-engine/bpmn-studio
# ⬆️Update Aurelia Packages

**Changes:**

1. Updates the aurelia-cli package.
2. Updates the aurelia-fetch-client package.
3. Updates the aurelia-tools package.
4. Updates the aurelia-templating-resources package.
5. Updates the aurelia-bootstrapper package.
6. Nails down the specific versions.

**Issues:**

Closes #1350 

PR: #PullRequest

## How can others test the changes?

1. Build and Start BPMN-Studio.
2. See that everything is working normally.

## PR-Checklist

Please check the boxes in this list after submitting your PR:
